rpcs3/util/fixed_typemap.hpp

```diff
@@ -10,6 +10,12 @@
 
 namespace stx
 {
+	template <typename T>
+	concept StopableThread = requires (T& t)
```

Comment:
Stopable -> Stoppable
StopableThread -> IsStoppable
Doesn't need to be in stx namespace also.

Reply:
Fixed.